git_machete/cli.py

```diff
@@ -647,8 +661,13 @@ def launch(orig_args: List[str]) -> None:
                 current_branch = git.get_current_branch()
                 machete_client.expect_in_managed_branches(current_branch)
                 machete_client.retarget_github_pr(current_branch)
+            elif github_subcommand == "sync":
+                machete_client.delete_unmanaged(opt_yes=False)
+                machete_client.checkout_github_prs(pr_nos=[],
```

Comment:
Hmmm `git machete github sync` fails if there are NO open pull requests authored by the current user:

```
$ git machete github sync 
Checking for unmanaged branches...
.............
Checking for open GitHub PRs...
Current user PawelLipski has no open pull request in repository foo/bar
```
(exit code 1)

Reply:
...this means that we don't even reach `delete_untracked`

Reply:
Actually, I think that `checkout_github_prs` should NOT fail if no PRs for the given criteria are found... just a message and exit code 0 is okay